### ISSUE TITLE
[Dialog] - Automated check failure of "An element of the given ControlType must support the text pattern"

### ISSUE THREAD

**Issue by Bhavana-Nimma** [microsoft/fluentui#18555]
### Environment Information

- **Package version(s)**: 8.18.0
- **Browser and OS versions**: Microsoft Edge, Windows 10

### Describe the issue:

In the page where dialog is opened, when automate checks are ran using "Accessibility insights for windows", there is a failure saying "An element of given ControlType must support the text pattern" for "document" section.

#### Actual behavior:

https://user-images.githubusercontent.com/85894656/121930918-92e57d80-cd60-11eb-9ac8-912ed4e1737a.mp4

#### Expected behavior:

This failure must be fixed

**Comment by gouttierre** [comment]
@Bhavana-Nimma - thank you for filing this issue and sharing the details into the problem. 

@ling1726 - I will assign this one for investigation with the accessibility label - however we may need to submit the issue to other portal for @smhigley.

**Comment by ling1726** [comment]
@Bhavana-Nimma  you are using accessibility insights for windows, which does not report accessibility problems for web. Please use accessibility insights for web when testing issues in web content
